src/Bicep.Core.Samples/Files/Modules_CRLF/main.json

```diff
@@ -2013,7 +2015,7 @@
               "version": "1.0.0"
             }
           },
-          "resources": [],
+          "resources": {},
```

Comment:
Wasn't expecting this to fix a bug, but here we are! Looks like we previously weren't correctly respecting the separate bicepconfig.json in the module directory for code generation.